client/components/activityform.jsx

```diff
@@ -40,24 +55,46 @@ export default class ActivityForm extends React.Component {
   handleSubmit(event) {
     const { activityName, details, planId } = this.state;
     event.preventDefault();
-    fetch('/api/activities/', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify({
-        activityName,
-        details,
-        planId
+    const requirement = window.location.hash.slice(14, 20);
```

Comment:
Instead of picking a spot to slice from the hash, you can check if App gave you `this.props.planId` or not.

https://github.com/mikesayala/final-project/blob/issue-3-user-can-update-plans/client/app.jsx#L33-L34